draft-ietf-rtcweb-security-arch.xml

```diff
@@ -556,6 +556,163 @@
+
+  a=identity:\
+    eyJpZHAiOnsiZG9tYWluIjoiZXhhbXBsZS5vcmciLCJwcm90b2NvbCI6ImJvZ3Vz\
+    In0sImFzc2VydGlvbiI6IntcImlkZW50aXR5XCI6XCJib2JAZXhhbXBsZS5vcmdc\
+    IixcImNvbnRlbnRzXCI6XCJhYmNkZWZnaGlqa2xtbm9wcXJzdHV2d3l6XCIsXCJz\
+    aWduYXR1cmVcIjpcIjAxMDIwMzA0MDUwNlwifSJ9
+      ]]></artwork>
+       </figure>
+       <t>
+         This specification does not define any extensions for the attribute
+       </t>
+       <t>
+         The identity-assertion value is a JSON <xref target="RFC8259"/> encoded string. The JSON object 
+         contains two keys: "assertion" and "idp". The "assertion" key value contains 
+         an opaque string that is consumed by the IdP. The "idp" key value contains a
+         dictionary with one or two further values that identify the IdP. See
+         <xref target="sec.request-assert"/> for more details.
+       </t>
+       <section title="Offer/Answer Considerations" anchor="sec.sdp-id-attr-oa">
+         <t>
+		   This section defines the SDP Offer/Answer <xref target="RFC6454"/> considerations for the SDP
+		   'identity' attribute.
+	     </t>
+	     <section title="Adding Identity Assertion to Session Description" anchor="sec.sdp-id-attr-oa-add">
+	       <t>
+             This section defines the procedures when an endpoint adds an identity assertion to
+             a session description within a session. This might take place when a session is initiated,
+             or later during the session.
+	       </t>
+           <section title="Offer Procedures" anchor="sec.sdp-id-attr-oa-add-off">
+             <t>
+               When an offerer sends an offer, in order to provide its identity assertion to the peer, it MUST include an 
+               'identity' attribute in the offer. In addition, it MUST include SDP 'fingerprint' attributes
+               for each fingerprint that have been used to assert the identity.
+             </t>
+             <t>
+               When the offerer receives an answer that contains an 'identity' attribute, the offerer can use
+               the attribute information to contact the IdP, and verify the identity of, the peer. If the offerer does not
+               accept the attribute (or the attribute value), or if the identity verification fails, it MUST 
+               take proper actions to reject the answer.
+             </t>
+           </section>
+           <section title="Answerer Procedures" anchor="sec.sdp-id-attr-oa-add-ans">
+             <t>
+               When an answerer receives in offer that contains an 'identity' attribute, the answerer can use the
+               the attribute information to contact the IdP, and verify the identity of, the peer. If the answerer 
+               does not accept the attribute value, or if the identity verification fails, it MUST reject the offer.
+             </t>
+             <t>
+               When an answerer sends an answer, in order to provide its identity assertion to the peer, it MUST include an 
```

Comment:
s/it MUST include/it includes/  This isn't an interop requirement.

Reply:
I assume you want the same change in the offerer procedures.